### PR TITLE
Show product detail

### PR DESCRIPTION
**Pull request by Baotranviet**

### CODE REVIEW

**Comment by Baotranviet** [comment]
ready

**Comment by quoctd-0607** [comment]
commented
conflict code

**Comment by Baotranviet** [comment]
ready

File: resources/views/products/detail.blade.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    <h4 class="mb-60">{{ $product->name }}</h4> <hr>
+                    <p  class="mb-20">{{ $product->description }}</p> <hr>
+                    <div class="price">
+                        <span>{{ number_format($product->price, 0, '', ',') }} đ</span> <hr>
+                    </div>
+                    <span>{{ trans('home.quantity')  }} {{ $product->quantity }}</span> <hr>
+                    <div class=" row product-caption align-items-center">
+                        <div class="col-lg-6 col-md-6">
+                            <div class="button-group-area mt-40">
+                                <a href="#" class="genric-btn success e-large">Add to Cart</a>
```
Comment (quoctd-0607) [inline resources/views/products/detail.blade.php]: i18n

File: resources/views/products/detail.blade.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                            </div>
+                        </div>
+                        <div class="col-lg-6 col-md-6 product-ratting mt-50">
+                            <i class="far fa-star"></i>
+                            <i class="far fa-star"></i>
+                            <i class="far fa-star"></i>
+                            <i class="far fa-star low-star"></i>
+                            <i class="far fa-star low-star"></i>
+                            <a href="#" class="genric-btn danger e-large ml-20">Vote</a>
```
Comment (quoctd-0607) [inline resources/views/products/detail.blade.php]: tương tự

**Comment by Baotranviet** [comment]
ready

**Pull request merged**